.github/workflows/ci-go.yaml

```diff
@@ -27,3 +27,16 @@ jobs:
           go-version: '1.21'
       - name: Run tests
         run: go test -race ./...
+  integration-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ariga/setup-atlas@v0
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+      - run: go install ./cmd/atlas-action
```

Comment:
This will be replaced with a pre-compiled binary once the release process is up.